styles/styles.css

```
@@ -57,29 +57,163 @@ body {
   font-size: 16px;
   font-weight: 700;
   line-height: 20px;
+  text-decoration: none;
   color: #2d3440;
-  border-radius: 16px;
+  border-radius: 26px;
+  outline: 4px solid rgb(247, 202, 214);
   background-color: #fcfcfc;
+  box-shadow: 0 4px 12px 0 rgba(45, 52, 64, .10);
+}
+
+.button-settings {
+  box-sizing: border-box;
+  padding: 8px 32px;
+  font-size: 14px;
+  font-weight: 700;
+  border: none;
+  border-radius: 20px;
+  outline: 2px solid #fcfcfc;
+  background: rgba(252, 252, 252, .30);
+}
+
+.button-settings:hover {
+  box-sizing: border-box;
+  border: none;
+  background: #fcfcfc;
+}
+
+.button-settings:active {
+  box-sizing: border-box;
+  border: none;
+  background: rgba(252, 252, 252, .50);
+}
+
+.button-settings:focus {
+  box-sizing: border-box;
+  border: none;
+  outline: 2px solid var(--basic-extra-dark, #2d3440);
+  background: rgba(252, 252, 252, .30);
 }
 
 .button-pink {
+  padding: 12px 32px;
+  font-family: Inter;
```

```suggestion
  font-family: inherit;
```

Поскольку это более надежно для изменения шрифта и плюс по критерию если уж используем название шрифта (кстати оно у нас в кавычках объявлено), то надо указывать семейство (как в `body`)

Но я советую `inherit`